scout/server/blueprints/variant/controllers.py

```diff
@@ -172,10 +171,8 @@ def variant(
     case_name,
     variant_id=None,
     variant_obj=None,
-    add_case=True,
     add_other=True,
     get_overlapping=True,
-    variant_category=None,
```

Comment:
This is redundant, since the category gets overwritten [later in the function](https://github.com/Clinical-Genomics/scout/blob/208da889b3a7fb70c4c3088b1400ad5fd422336a/scout/server/blueprints/variant/controllers.py#L232)